icu4j/main/classes/core/src/com/ibm/icu/impl/units/UnitsConverter.java

```diff
@@ -11,12 +11,12 @@
 
 import com.ibm.icu.util.MeasureUnit;
 
-public class UnitConverter {
+public class UnitsConverter {
     private BigDecimal conversionRate;
     private BigDecimal offset;
 
     /**
-     * Constructor of `UnitConverter`.
+     * Constructor of <code>UnitsConverter</code>.
      * NOTE:
      * - source and target must be under the same category
      * - e.g. meter to mile --> both of them are length units.
```

Comment:
Doesn't look like this javadoc gets rendered to html, so this probably doesn't matter / is entirely optional:

A list in javadoc probably needs to be done via html <ul> / <li> rather than Markdown syntax. AFAIK.